python/doc/user_manual/graphs.rst

```diff
@@ -48,3 +48,5 @@ Graphical visualisation object
 
     :template: function.rst_t
     viewer.PlotDesign
+    viewer.ToImageString
```

Comment:
ToImageString is internal and should not be documented, perhaps it should be marked as such, usually in Python private methods begin with an underscore

Reply:
There is an issue here. The ToImageString is used in clients of the viewer module, such as Graph, Drawable and DrawableImplementation classes:

https://github.com/openturns/openturns/blob/96543814e19406e169727f9c08d2bcb77b2d0b95/python/src/Drawable.i#L21

and 

https://github.com/openturns/openturns/blob/96543814e19406e169727f9c08d2bcb77b2d0b95/python/src/Graph.i#L48

https://github.com/openturns/openturns/blob/96543814e19406e169727f9c08d2bcb77b2d0b95/python/src/DrawableImplementation.i#L16

Moreover, the method does not have a prepending underscore. 

Hence it is technically not private at this time. It perhaps should be, but this is not a documentation issue. Do you suggest to : rename it ? remove it from the graph.rst document ? Both ?

Reply:
I renamed ToImageString into _ToImageString and removed into from the doc.